AUTHORS.rst

```diff
@@ -0,0 +1,29 @@
+..
+    This file is part of Invenio-JSONSchemas.
+    Copyright (C) 2015 CERN.
+
+    Invenio is free software; you can redistribute it
```

Comment:
This is a bit inconsistent with https://github.com/inveniosoftware/invenio-jsonschemas/pull/1/files#diff-830968569b50f87c82e22a0042a7f760R2.